.gitignore

```diff
@@ -1,3 +1,4 @@
+.idea
```

Comment:
Do we want to have IDE files/dirs in the local `.gitignore` instead of leaving it up to the user to add them to their global gitignore?

Reply:
If the local user hasn't added it to their global gitignore then we risk having it end up in the repo, better safe than sorry as it doesn't hurt